x-pack/plugins/canvas/server/routes/shareables/download.test.ts

```diff
@@ -4,17 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-jest.mock('fs');
```

Comment:
It's probably just me who doesn't see the big picture, but the changes to this file doesn't seem to be related to the PR?

Reply:
Packages are under `packages/` in development and under node_modules in production. Because of this, we can't rely on a relative position for `REPO_ROOT` and are walking up the tree. [Here ](https://github.com/elastic/kibana/blob/f35cb32012cb09c46ec8b7b1759004554a7c3e8e/packages/kbn-utils/src/repo_root.ts) is the file moved from dev-utils. This broke where tests were mocking all `fs` calls.